inc/spbc-settings.php

```diff
@@ -3394,13 +3424,15 @@ function spbc_list_table__get_args_by_type($table_type)
                 $accordion_default_args,
                 array(
                     'func_data_prepare' => 'spbc_field_scanner__prepare_data__files',
+                    //'func_data_prepare' => 'spbc_field_scanner__approved__data_prepare',
                     'columns'           => array(
                         'cb'         => array('heading' => '<input type=checkbox>', 'class' => 'check-column',  'width_percent' => 2),
                         'path'       => array('heading' => 'Path', 'primary' => true, 'width_percent' => 40),
                         'weak_spots' => array('heading' => 'Detected', 'width_percent' => 20),
                         'size'       => array('heading' => 'Size, bytes', 'width_percent' => 7),
                         'perms'      => array('heading' => 'Permissions', 'width_percent' => 7),
                         'mtime'      => array('heading' => 'Last Modified', 'width_percent' => 7),
+                        'status'     => array('heading' => 'Approved by', 'width_percent' => 15),
```

Comment:
I see you have added a new column, but summary all columns width will be more than was before. Make sure that table shown properly.

Reply:
Sum width = 98%